clang/docs/ReleaseNotes.rst

```diff
@@ -1119,6 +1119,14 @@ Windows Support
 
 LoongArch Support
 ^^^^^^^^^^^^^^^^^
+- Added a code model attribute for the global variable. The following values
```

Comment:
"The `model` attribute is now supported for overriding the default code model used to access global variables." or something like that?

Reply:
Thanks.